dataset_configs/portuguese/coraa/config.yaml

```diff
@@ -0,0 +1,83 @@
+documentation: |
+  Coraa Portuguese
+  ###########
+
+  The config  performs the following data processing.
+
+  1. Downloads and extracts all the data from the "https://huggingface.co/datasets/gabrielrstan/CORAA-v1.1/tree/main"
+  2. Replaces certain non-supported characters, abbreviations and  punctuation marks with equivalent supported versions.
+  4. Drops any data that contains high/low character occurence.
+  5. Drops any data that contains symbols not in the supported alphabet.
+
+  **Required arguments**.
+
+  * **workspace_dir**: specify the workspace folder where all audio files will be stored.
+  * **data_split**: should be "train", "dev" or "test".
+
+  Note that you can customize any part of this config either directly or from command-line.
+  Here are some common customizations to consider:
+
+  **Output format**.
+
+  This config dumps the final manifest at ``${workspace_dir}/${data_split}_manifest.json``.
+  The output manifest contains the following fields:
+
+  * **audio_filepath (str)**: relative path to the audio files.
+  * **text (str)**: transcription, including punctuation ".,?" and capitalization.
+  * **duration (float)**: audio duration in seconds.
+  
+
+processors_to_run: "1:"
+workspace_dir: ??
```

Comment:
Does 2 question marks work? Lets stick to 3 (`???`) like in the other configs and like mentioned in the documentation https://omegaconf.readthedocs.io/en/2.3_branch/usage.html#mandatory-values

Reply:
Done